darwin/client.py

```diff
@@ -84,12 +84,12 @@ def get(
     def put(
         self,
         endpoint: str,
-        payload: Dict,
+        payload: Dict[Any, Any],
```

Comment:
Can't this be turned at least into:
```suggestion
        payload: Dict[str, Any],
```

Same for other `payload` arguments down below